doc/radosgw/multisite.rst

```diff
@@ -1688,3 +1688,98 @@ On any cluster in the realm:
 .. _`Sync Policy Config`: ../multisite-sync-policy
 .. _`Server-Side Encryption`: ../encryption
 .. _`Compression`: ../compression
+
+S3-style subdomains for buckets
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Ceph Object Gateway supports S3-style subdomains, which provides
+virtual-host-style-URLs for the buckets. Add wildcard entry with server's IP
+address to your DNS server and configure DNS for this entry.
+
+Add hostname in the Ceph Object Gateway zone group:
+
+#. Get the zone group:
+
+   .. prompt:: bash $
+
+      radosgw-admin zonegroup get --rgw-zonegroup={zonegroup-name}
+
+#. Take back up of the zone group configuration:
+
+   .. prompt:: bash $
+
+      radosgw-admin zonegroup get --rgw-zonegroup={zonegroup-name} > zonegroup.json
+      cp zonegroup.json zonegroup.back.json
```

Comment:
That would also make Rook's implementation easier in the long run, but we would need to either make it so that the feature only works for Ceph versions that have the `--hostnames` flag present, or use both implementations in Rook to start.

Reply:
for rook, you might consider switching to the rgw_dns_name version instead

Reply:
> can we add a `--hostnames` option to the `zonegroup modify` command to avoid messing with json here?

This one was not working AFAIR, I did test it on master(a month ago) don't have the setup atm. In v18.2.6 it is throwing error. Not listed in help as well 
```bash
#../src/mrun c1 radosgw-admin zonegroup -h | grep host
# ../src/mrun c1 radosgw-admin -v
ceph version 18.2.0-212-g6d9f08b2cba (6d9f08b2cbaeb750920d27bb8a3bf7e5f574009d) reef (stable)
# ../src/mrun c1 radosgw-admin zonegroup modify --rgw-zonegroup=us --hostnames "http://localhost:8001" --debug-rgw=20
2023-11-16T05:27:54.304-0500 7fd6b6040780 -1 WARNING: all dangerous and experimental features are enabled.
2023-11-16T05:27:54.307-0500 7fd6b6040780 -1 WARNING: all dangerous and experimental features are enabled.
ERROR: invalid flag --hostnames
```